### PR TITLE
Only use the test runner plugin when CI is set in the env

### PR DESCRIPTION
**Pull request by superm1**
This lets the tests be installed onto a system without FakeDevices showing up.

### CODE REVIEW

File: plugins/test/fu-plugin-test.c [hunk 1]
```diff
@@ -45,6 +45,13 @@ fu_plugin_destroy (FuPlugin *plugin)
 gboolean
 fu_plugin_startup (FuPlugin *plugin, GError **error)
 {
+	if (g_getenv ("FWUPD_TESTS") == NULL) {
+                g_set_error (error,
+                             FWUPD_ERROR,
+                             FWUPD_ERROR_NOT_FOUND,
+                             "Test plugin is only used for continuous integration.");
```
Comment (hughsie) [inline plugins/test/fu-plugin-test.c]: Tabs here please, and no need for a full stop in a GError message.

**Comment by hughsie** [comment]
With the two little fixes, LGTM thanks.

File: plugins/test/fu-plugin-test.c (hunk 1 above)
Comment (superm1) [inline plugins/test/fu-plugin-test.c]: tabs/spaces were a casualty of copy paste, i'll fix that thanks for catching.
I'm unsure what you mean by full stop though, can you clarify?

Comment (hughsie) [inline plugins/test/fu-plugin-test.c]: `Test plugin is only used for continuous integration.` -> no full stop

**Pull request merged**